[DOCS-6111] AIS1.3 (#192)
Alfresco/docs-alfresco: intelligence-services/latest/config/comprehend.md

### Digital Workspace

The Digital Workspace configuration for custom AI requires modification of an existing configuration file (`ai-view.extension.json`). The JSON file is included in the Intelligence Services distribution zip. This is unlike the repository and Share configuration, where only new files are created and mounted in the containers.

#### App extension

File name: `ai-view.extension.json`

Mount location and example:

```bash

> ./ai-view.extension.json:/usr/share/nginx/html/assets/ai-view.extension.json

Reviewer: ```suggestion
./ai-view.extension.json:/usr/share/nginx/html/assets/plugins/ai-view.extension.json
```

### Digital Workspace - Docker service definition

```yaml
  digital-workspace:
    image: quay.io/alfresco/alfresco-digital-workspace:2.1.0-adw
    environment:
      BASE_PATH: ./
      APP_CONFIG_PLUGIN_AI_SERVICE: "true"
    volumes:
      - ./ai-view-extension.json:/usr/share/nginx/html/assets/plugins/ai-view-extension.json
```

> In the above `docker-compose` snippet, the modified `ai-view-extension.json` configuration file must be mounted in the Digital Workspace container. The environment variable `APP_CONFIG_PLUGIN_AI_SERVICE:` when set to `true` allows the Digital Workspace to generate transcripts for audio and video files, including indexing and metadata generation.

Reviewer: ```suggestion
In the above `docker-compose` snippet, the modified `ai-view-extension.json` configuration file must be mounted in the Digital Workspace container. The environment variable `APP_CONFIG_PLUGIN_AI_SERVICE:` when set to `true` allows the Digital Workspace to index and search by the content of the transcripted files and scanned images.
```

> In the above `docker-compose` snippet, the modified `ai-view-extension.json` configuration file must be mounted in the Digital Workspace container instead of the standard one (from the Content Services deployment). The environment variable `APP_CONFIG_PLUGIN_AI_SERVICE:` when set to `true` allows the Digital Workspace to generate transcripts for audio and video files, including indexing and metadata generation.

Reviewer: ```suggestion
In the above `docker-compose` snippet, the modified `ai-view-extension.json` configuration file must be mounted in the Digital Workspace container. The environment variable `APP_CONFIG_PLUGIN_AI_SERVICE:` when set to `true` allows the Digital Workspace to generate transcripts for audio and video files, including indexing and metadata generation.
```
In this new version we are not replacing the original, we are just adding this file
